lib/utils.js

```diff
@@ -7,98 +10,79 @@ var assert = require("assert");
  * @returns Configuration Object
  */
 exports.parseMongoConnectionURL = function (mongoConnectionURL) {
-  var result = {};
 
-  //The connection string will be of the form mongodb://user:password@host:port,host2port2/databasename?option=someOption
+  //The connection string will be of the form mongodb://username:pass@db1.prv.mbaas2.cluster.feedhenry.net,db2.prv.mbaas2.cluster.feedhenry.net:27017/domain-vvuq7douloe2zkyktub7fcmt-demos-dev?replicaSet=cluster-mbaas2_rs1
   //String to break down the mongodb url into a ditch config hash
-  var auth_hosts_path_options = new Buffer(mongoConnectionURL).toString().split("//")[1];
+  const parts = mongoConnectionURL.split(',');
 
-  //user:password , host:port,host2:port2/databasename?option=someOption
-  var auth = auth_hosts_path_options.split("@")[0];
+  // The name comes from the path in the URL. We must always use the last host
+  function getName () {
+    const pathname = url.parse(parts[parts.length - 1]).pathname;
```

Comment:
Did you validate with someone that knows a lot about it if has some reason for before don't take just the last one? Will it work well expected of has just one? Did you test uses of fh.db with a locally DB and in a project without upgraded the database? 

What will be the behaviour if the last host is down? Will it automatically redirect for the another?